src/app.ts

```diff
@@ -112,6 +114,27 @@ export const createApp = (repo: ArticleRepository, config: Record<string, string
     }
   });
 
+  app.post('/import-version', async (req, res, next) => {
+    try {
+      const data = req.body as EnhancedArticle;
```

Comment:
@will-byrne Actually, I just thought that really we should validate this input. Any thoughts on how we go about that?

Reply:
something like joi could be used, there may also be a way of doing it through express and generics but id have to experiment.